grr/server/grr_response_server/gui/ui/components/flow_details/helpers/osquery_results_table_test.ts

```diff
@@ -1,8 +1,24 @@
 import { TestBed, async } from '@angular/core/testing';
 import { HelpersModule } from './module';
-import { OsqueryTableBuilder, ParsedDetailsTable } from './osquery_test_utils';
 import { DebugElement } from '@angular/core';
 import { OsqueryResultsTable } from './osquery_results_table';
+import { By } from '@angular/platform-browser';
+import { OsqueryTable } from '@app/lib/api/api_interfaces';
+import { newOsqueryTable } from '../plugins/osquery_details_test';
+
+/** Helper data structure to parse an osquery_results_table */
+class OsqueryResultsTableDOM {
+  queryDiv = this.rootElement.query(By.css('.results-query-text'));
```

Comment:
Generally use `readonly` if possible. Restricting mutability is desirable to make code easier to understand.